### ISSUE TITLE
Cant reconnect after deinstall IOS App

### ISSUE THREAD

**Issue by Allgamer93** [functionland/fx-components#390]
I use the CM4 Box. I install the latest image and connect with the IOS Tablet. everything perfect. After them the App shows "disconnect" to Box. Then i deinstall the IOS App. Now i want to reconnect to existing blox ,it doenst show the Box. But in Wifi settings i see  that bos is connect to my router

**Comment by mrodriguez3313** [comment]
Hi All Gamer! Thank you for this issue!

So, if I understand correctly, you're using your iOS tablet to connect the blox to your home wifi. After you uninstall the app you want to reconnect to the same Blox?

To do this, you should be able to press on the "Connect to Existing Blox" And the blox should show up. If not give the Blox a reboot (unplug/plug it in). It should show up because, as you mentioned, you can see the Blox in your wifi settings.

**Comment by Allgamer93** [comment]
yes, i try it a lot, but it doenst work. Same if i try to find with my iphone with the blox app.

**Comment by Allgamer93** [comment]
it doenst show the exisiting blox. It not possible to refresh. After replug the blox shows light withe and green and nothing

**Comment by mrodriguez3313** [comment]
Last thing, just make sure your phone is on the same network you connected your Blox to. Also, make sure to completely force quit the app, or try uninstalling/reinstalling the app again to delete all data. 

That being said, sometimes a couple power cycles will fix the issue. So make sure to disconnect the Blox for a couple seconds and turn it back on. 

> After replug the blox shows light withe and green and nothing

This is the expected behavior of a Blox properly connected to the wifi.
